src/components/Create/components/RewardsList/AddEditRewardModal.tsx

```diff
@@ -236,19 +282,108 @@ export const AddEditRewardModal = ({
             )}
           </Space>
         </Form.Item>
-        <Form.Item
-          name="externalUrl"
-          label={t`External link`}
-          rules={[
-            inputIsValidUrlRule({
-              label: t`External link`,
-              prefix: 'https://',
-              validateTrigger: 'onCreate',
-            }),
-          ]}
+        <CreateCollapse
+          activeKey={advancedOptionsOpen ? ['0'] : []}
+          onChange={onCollapseChanged}
         >
-          <PrefixedInput prefix="https://" />
-        </Form.Item>
+          <CreateCollapse.Panel
+            header={<OptionalHeader header={t`Advanced options`} />}
+            key={0}
+            hideDivider
+          >
+            <Form.Item
+              extra={
+                <span className="text-sm">
+                  <Trans>Set aside a percentage of freshly minted NFTs.</Trans>
+                </span>
+              }
+            >
+              <JuiceSwitch
+                value={isReservingNfts}
+                onChange={setIsReservingNfts}
+                label={
+                  <span className="flex items-center gap-2">
+                    <Trans>Reserve NFTs</Trans>
+                    <Tooltip
+                      className="cursor-help text-grey-500 dark:text-grey-300"
+                      title={t`If Reserved NFTs are set, the first reserved NFT from the tier will be distributable once the tier receives its first mint.`}
+                    >
+                      <QuestionCircleOutlined />
+                    </Tooltip>
+                  </span>
+                }
+              />
+            </Form.Item>
+            {isReservingNfts && (
+              <>
+                <span className="mb-4 flex flex-col gap-2 whitespace-nowrap md:flex-row md:items-center">
+                  <Trans>Reserve 1 NFT of every</Trans>
+                  <div className="max-w-[78px]">
+                    <Form.Item
+                      className="mb-0"
+                      name="nftReservedRate"
+                      rules={[
+                        inputMustExistRule({ label: t`Reserved rate` }),
+                        inputIsIntegerRule({
+                          stringOkay: true,
+                        }),
+                      ]}
+                    >
+                      <FormattedNumberInput min={2} placeholder="0" />
+                    </Form.Item>
+                  </div>{' '}
+                  <Trans>NFTs minted for:</Trans>
+                </span>
+                <Form.Item
+                  className="mb-8"
+                  name="beneficiary"
+                  label={t`Ethereum wallet address`}
```

Comment:
<img width="530" alt="Screen Shot 2023-01-24 at 2 26 40 pm" src="https://user-images.githubusercontent.com/96150256/214212131-a62ef4aa-ed82-4ab5-bf45-601cd969fda1.png">

Error message isn't going away for me here after typing in an ENS

Reply:
These are are quirks (and annoyances) of antd and the way that the address input code works. Probably out of scope as these are larger problems in our code :(

Reply:
Actually, it looks like the problem you're facing might be different re invalid name even after entering. I'll see if I can recreate

Reply:
Are you entering it on goerli? johnnyd.eth isn't registered on goerli so this is expected behaviour

Reply:
Ahh yes good 

> Are you entering it on goerli? johnnyd.eth isn't registered on goerli so this is expected behaviour

Ahh yes good point sir, would be that.